lib/mongoose_push.ex

```diff
@@ -72,7 +72,16 @@ defmodule MongoosePush do
 
     {time, push_result} =
       if pool == nil do
-        Logger.error(~s"No pool matching mode=#{mode} and tags=#{inspect(tags)}")
+        Logger.error(
+          fn ->
```

Comment:
Do we really need this function? This is error log, it's very unusual for them to compiled-out.